fastfield_codecs/src/frame_of_reference.rs

```diff
@@ -0,0 +1,303 @@
+use crate::FastFieldCodecReader;
+use crate::FastFieldCodecSerializer;
+use crate::FastFieldDataAccess;
+use crate::FastFieldStats;
+use std::io::{self, Read, Write};
+use tantivy_bitpacker::compute_num_bits;
+use tantivy_bitpacker::BitPacker;
+
+use common::BinarySerializable;
+use common::DeserializeFrom;
+use tantivy_bitpacker::BitUnpacker;
+
+const BLOCK_SIZE: u64 = 128;
+
+#[derive(Clone)]
+pub struct FrameOfReferenceFastFieldReader {
```

Comment:
I think we can use the abbreviation `FOR` for once :)
It's one of those acronym that is more famous than the underlying expression.